bootstrap.php

```diff
@@ -55,3 +55,47 @@
         'user_id'   => Cookie::get('logged_user_id'),
     ));
 });
+
+\Event::register_function('admin.beforeLogin', function ($params) {
+    $controller = \Arr::get($params, 'controllerInstance');
+    $wait_failures_config = \Arr::get(\Config::load('novius_loginhistory::config', true), 'wait_after_admin_login_failures', []);
+    $wait_failures_enabled = \Arr::get($wait_failures_config, 'enabled', true);
+    $callback_is_whitelisted = \Arr::get($wait_failures_config, 'is_whitelisted');
+
+    if ($wait_failures_enabled && is_callable($callback_is_whitelisted) && !$callback_is_whitelisted()) {
```

Comment:
if `$callback_is_whitelisted` is not callable, you have to do this test. I think a better condition is:

```
if ($wait_failures_enabled && (!is_callable($callback_is_whitelisted) || !$callback_is_whitelisted())) {
```